store/sqlstore/post_store.go

```diff
@@ -745,9 +807,15 @@ func (s *SqlPostStore) GetPostAfterTime(channelId string, time int64) (*model.Po
 	return post, nil
 }
 
-func (s *SqlPostStore) getRootPosts(channelId string, offset int, limit int) ([]*model.Post, *model.AppError) {
+func (s *SqlPostStore) getRootPosts(channelId string, offset int, limit int, skipRootFetch bool) ([]*model.Post, *model.AppError) {
```

Comment:
As an aside, `getRootPosts` makes no sense for this function name -- this just gets all posts, right?